### DISCUSSION TITLE
Replaying events from existing eventstore to create new Projections

### DISCUSSION THREAD

**Discussion by InfantIvan** [rasmus/EventFlow#904] [Q&A] [answered]
First of all my hearty thanks to Rasmus and other moderators. We are using eventflow in production systems and it has exceeded all our expectations.

Would like to implement replaying events to create new projections now. Could you please point me to some reference articles or samples?

I tried to find in your documentation( https://docs.geteventflow.net/) but couldn't find anything specific to replaying events

**Comment by rasmus** [comment, marked as answer]
Sorry for the late reply.

When you say "replay events", do you want to send them to read models, aggregates or subscribers? Since ever part of EventFlow is built in very small isolated classes, you could build something yourself by e.g. getting events from `IEventStore` and then have a look at the `DomainEventPublisher` implementation on how events are propagated to the different parts. It should be relatively strait forward

**Reply by InfantIvan** [reply]
@rasmus . Yes.. I figured it out. As you said it was quite straight forward

**Reply by jessemillerrev** [reply]
Is this not something built into Eventflow? The ability to rebuild your read models or add new ones and have them be populated by all historical events?
